src/vs/editor/contrib/codeAction/browser/codeActionMenu.ts

```diff
@@ -632,24 +713,16 @@ export class CodeActionMenu extends Disposable implements IEditorContribution {
 		const menuActions = this.getMenuActions(trigger, actionsToShow, codeActions.documentation);
 
 		const anchor = Position.isIPosition(at) ? this._toCoords(at) : at || { x: 0, y: 0 };
+
+		const params = <ICodeActionMenuParameters>{ options, trigger, codeActions, anchor, menuActions, context: this._contextViewService, editor: this._editor, showDisabled: true, visible: this._visible, menuObj: this };
```

Comment:
You should not need to pass through `context ` or `editor`  as you can access these on `this.`